### PR TITLE
[READY]Buffs Teslium Explosions

### PR DESCRIPTION
**Pull request by silicons**
Now there's a .5 second delay before the initial shock instead of 1 second, but the follow up shocks are 1.5 second delay each.
It was neigh-useless as you can easily run out of range with 1 second of warning, .5 second still allows for quick reflex people to run but it's harder.

### CODE REVIEW

**Comment by XDTM** [comment]
It looks like if you have less than 40u of explosion you get to wait 3.5 seconds for the first shock. Maybe you could move the sleep(15) into the if(..)?

**Comment by kevinz000** [comment]
Hm, should fix it

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -375,16 +375,16 @@
...
 	var/T2 = created_volume * 50
 	var/T3 = created_volume * 120
-	sleep(10)
-	if(created_volume >= 75)			//10 units minimum for lightning, 40 units for secondary blast, 75 units for tertiary blast.
+	sleep(5)
+	if(created_volume >= 75)
 		tesla_zap(holder.my_atom, 7, T1)
 		playsound(holder.my_atom, 'sound/machines/defib_zap.ogg', 50, 1)
-	sleep(10)
+		sleep(15)
```
Comment (optimumtact) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: why did you indent the sleep

**Comment by KorPhaeron** [comment]
@kevinz000

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 1 above)
Comment (XDTM) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: so you don't have to wait 3.5 seconds for it to explode if you have less than 75 teslium

**Pull request merged**